### PR TITLE
DEV-2729: Airpods starve on startup

### PR DESCRIPTION
**Pull request by amerhifi**
https://highfidelity.atlassian.net/browse/DEV-2729

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Can someone from High Fidelity approve this PR?

File: libraries/audio-client/src/AudioClient.cpp [hunk 1]
```diff
@@ -335,9 +335,9 @@ AudioClient::AudioClient() {
     connect(&_receivedAudioStream, &InboundAudioStream::mismatchedAudioCodec, this, &AudioClient::handleMismatchAudioFormat);
 
     // initialize wasapi; if getAvailableDevices is called from the CheckDevicesThread before this, it will crash
-    getAvailableDevices(QAudio::AudioInput, QString());
-    getAvailableDevices(QAudio::AudioOutput, QString());
-    
+    defaultAudioDeviceName(QAudio::AudioInput);
```
Comment (amerhifi) [inline libraries/audio-client/src/AudioClient.cpp]: wasapi is initialized during the call to getting the device name, no need to run the expensive getavailabledevices

File: libraries/audio-client/src/AudioClient.cpp [hunk 2]
```diff
@@ -787,8 +787,11 @@ void AudioClient::start() {
         inputName = _hmdInputName;
         outputName = _hmdOutputName;
     }
+    
+    //initialize input to the dummy device to prevent starves
+    switchInputToAudioDevice(HifiAudioDeviceInfo());
```
Comment (amerhifi) [inline libraries/audio-client/src/AudioClient.cpp]: initializing a dummy input until the real switch happens on first audio detection prevents starve as well as works around the issue of selecting same device twice.  Airpods at this time freeze the audio thread if selected twice as input. Output does not have this issue

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/22739/HighFidelity-Beta-PR16550-e02725e.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/22739/HighFidelity-Beta-Interface-PR16550-e02725e.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/22739/HighFidelity-Beta-PR16550-e02725e.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/22739/HighFidelity-Quest-Beta-PR16550-e02725e.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/22739/HighFidelity-Beta-PR16550-e02725e.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/22739/HighFidelity-Beta-Interface-PR16550-e02725e.exe

**Pull request merged**